plugins/ruby-foreman-tasks/debian/control

```diff
@@ -2,7 +2,7 @@ Source: ruby-foreman-tasks
 Section: ruby
 Priority: extra
 Maintainer: Michael Moll <mmoll@mmoll.at>
-Build-Depends: debhelper, git | git-core, foreman (>= 2.6.0~rc1), foreman-nulldb (>= 2.1.0~rc1), foreman-assets, python-dev
+Build-Depends: debhelper, git | git-core, foreman (>= 2.6.0~rc1), foreman-nulldb (>= 2.1.0~rc1), foreman-assets, python-dev, bundler
```

Comment:
That's a cheap workaround for now, will be dropped in the big "plugin.mk" change again